libs/challenge-registry/user-profile/src/lib/user-profile-stats/_user-profile-stats-theme.scss

```diff
@@ -62,8 +64,8 @@
     border-color: transparent;
   }
   .basic-stats-frameorganizations {
-    border-color: $dl-color-default-primary2;
-    background-color: rgba(255, 255, 255, 1);
+    border: solid 1px $shadow;
```

Comment:
is it possible to move the 
- `border: solid 1px` to "user-profile-stats.scss"
-  keep `border-color: $shadow` in `_user-profile-stats-theme.scss` where it only collects the colors.